masonite/view.py

```diff
@@ -12,27 +16,47 @@ def view(template='index', dictionary={}):
 
 
 class View(object):
+    """
+    Render template view
+    """
 
     def __init__(self):
         self.dictionary = {}
         self.composers = {}
+        self.container = None
+        self.cache_time = None
+        self.cache_type = None
+        self.template = None
         self.env = Environment(
             loader=PackageLoader('resources', 'templates'),
             autoescape=select_autoescape(['html', 'xml'])
         )
 
+    def load_container(self, container):
+        self.container = container
+        return self
+
     def render(self, template, dictionary={}):
         self.dictionary.update(dictionary)
 
+        # Load template
+        self.template = template
+
+        # Check if use cache and return template from cache if exists
+        if self.__cached_template_exists() and not self.__is_expired_cache():
```

Comment:
the `and not self.__is_expired_cache()` logic is reverse. Currently the template will keep on rendering the same cached template over and over and won't actually render and cache the new template when it expires but for some reason switching it to `and self.__is_expired_cache()` made it work like it should.

Reply:
The name of the function is ok, the problem is that the function is returned upside down. Is not it better to change the return?

Reply:
yeah change the return values

Reply:
the logic of `and not self.__is_expired_cache()` looks good you just have to flip the return values

Reply:
Great!